rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+# Gruntwork.io SEO RFC
+
+I read a lot of material in SEO, including going through a [guide to SEO](https://moz.com/beginners-guide-to-seo/).
+
+## Important take aways
+
+- We currently have some broken stuff in our website that should take priority over trying to improve SEO by creating more content. Ranked by (my perceived) importance:
+  - Searching for site:gruntwork.io/guides will show us that our content is duplicated between the www and non www versions of the website. This is diluting our ranking. We need to add a canonical link to our headers
+  - Our posts don't have meta descriptions
+  - Our internal links usually have the same anchor text (the title of the guide), but Google interprets that as not being natural and will penalize us for that
+  - We don't have a [sitemap](https://support.google.com/webmasters/answer/183668?hl=en)
+- Many websites use rel=nofollow (blocks crawling and backlink ranking bonus) on all external content or on all of their comment section. This is so because linking to bad quality content penalizes their own ranking. We need to verify if it's the case before engaging in any comment marketing for SEO reasons since this nullifies SEO advantages (but might still be useful for driving users that will click on it). Backlinks are also more important from topically similar websites. Link in hashicorp >>>> link in reddit.
```

Comment:
Yea, rel=nofollow is definitely common... Not sure if there's an easy way to add that for links created via Markdown... Perhaps punt on that until we migrate the site to SaaS?

Reply:
Markdown accepts plain html just fine. We could just use the old fashioned `<a>` but yes, I'm not sure how much of a priority this is. It requires a bit more of research to understand the depth of the problem. But it's good to be aware. I didn't about link equity before so I really wanted to call it out.